<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DadosDeVenda</class>
 <widget class="QDialog" name="DadosDeVenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="IdGerente_txt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Id Gerente:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Senha_txt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Senha Gerente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dados de Caixa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Lucro Total</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lucro por dia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lucro por Funcionario</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lucro por Funcionario por dia</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="ConfirmarButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Forma de analise:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Id Do Funcionario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dia em Quest√£o:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdFuncionario_txt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>301</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Dia_txt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>301</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
